### ISSUE TITLE
[Request] Create PowerCLI shortcut post-deployment

### ISSUE THREAD

**Issue by virtualex-itv** [bcurran3/ChocolateyPackages_CCR#32]
I'd like to request that after modules are installed, and additional command be executed to create a shortcut on the public profile's desktop which, when double-clicked, launches a PowerShell session and imports the module so that it behaves the same way the legacy ".exe" installer did.  I've done some testing and got this to work in my test environment.

Additionally, the dependency for PowerShell ≥ 5.0 fails as no PowerShell "5.0" version is listed in the chocolatey package repo. This would need to be changed to "5.0.10586.20161201" to work correctly.

![image](https://user-images.githubusercontent.com/12056704/41204241-f8e90760-6caf-11e8-87c7-3630b681f3f4.png)

**Comment by bcurran3** [comment]
RE: PowerShell, dependency is set to the powershell package to be equal to or greater than 5.0, the current package is 5.1.14409.20180105 . This should all work. Have you experienced problems or is this a guess?

RE: shortcut - can you supply the shortcut parameters please?

**Comment by virtualex-itv** [comment]
RE: PowerShell dependency, it gave me errors in the test environment with the version specified as 5.0 since it's checking chocolatey repo for a matching version.  It would be best to remove the version and just call the "PowerShell" dependency which will install the latest 5.1.14409.20180105 package.

RE: shortcut, I can but would rather send my changes via a pull request and/or be made a maintainer (preferred).  I wouldn't make any revisions without consulting you first.  Let me know how you'd like to proceed and feel free to reach me on Gitter.

**Comment by bcurran3** [comment]
I still don't understand why you had problems with the PowerShell dependency. I've got a feeling that your test environment is PS4 and your bumping into the problem mentioned in the PACKAGE NOTES: https://github.com/bcurran3/ChocolateyPackages/blob/master/vmware-powercli-psmodule/readme.md

Feel free to do a PR.

**Comment by bcurran3** [comment]
UPDATE: I see the shortcut you like in your repo: https://github.com/virtualex-itv/chocolatey/blob/master/_packages/vmware-powercli-psmodule/tools/ChocolateyInstall.ps1. This doesn't really need a PR, it's simple enough for me to add later.

**Comment by virtualex-itv** [comment]
The test environment I used to test the deployment leverages VirtualBox and Vagrant which is the exact same method/environment that the chocolatey moderators use when they test.  In this environment, the dependency would error since there is no PowerShell package with a simple "5.0" version listed in their repo.  If you removed the version it would work as it would then just use the "PowerShell" package which is currently at the 5.1.x version I mentioned previously.  I am making some changes to the way I created the icon since there is a native way to do it without using my function.  I will test it out and submit you a PR which hopefully you'll consider and use.  Again I'd love to help maintain this package as I am an avid user of PowerCLI and a VMware EE. :)  Trying to figure out ways to get more of our apps available on chocolatey.
